requirements/default.txt

```diff
@@ -1,6 +1,10 @@
 oemof.solph>=0.4.1
 pandas>=0.24.0,<=1.0.5
+pyomo!=5.7.3 # version 5.7.3 make mvs very slow
```

Comment:
Why not define `pyomo <= 5.7.3`? I would assume that further updated versions have the same problem?

Reply:
I would not assume this, for example pandas version 1.1.0, 1.1.1 and 1.1.2 had problems with mvs, but newer versions don't

I'd rather be notified that there is a problem when pyomo 5.7.4 comes out and adapt the requirements then rather than missing out on updates